test.rb

```diff
@@ -0,0 +1,114 @@
+require_relative 'enumerable'
+
+my_array = [1, 2, 3, 4, 5]
+my_hash = { x: 1, y: 2, z: 3 }
+my_text = %w[ant bear cat]
+arg = proc { |value| value * 2 }
+
+puts '==========================this is my each method'
+p my_array.my_each
+p(my_hash.my_each { |value| p value })
+p((1..10).my_each { |value| p value })
+# puts '==========================this is  each method'
+# p my_array.each
+# p my_hash.each { |value| p value }
+# p (1..10).each { |value| p value }
```

Comment:
You ought not to have commented out code is committed. If you need to review a previous version of your code, use git history.

See this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from our general requirements.

Reply:
You can consider also not committing/tracking test.rb altogether if it's for your personal use.